app/helpers/mortgage_calculator/application_helper.rb

```diff
@@ -12,25 +12,25 @@ def calculator_name
 
     def full_mortgage_calculator_url
       if I18n.locale == :cy
-        "https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-morgais"
+        "https://www.moneyhelper.org.uk/cy/homes/buying-a-home/mortgage-calculator"
```

Comment:
no, this was some kind of weird blanket thing that they did to change moneyadviceservice to moneyhelper but they are completely wrong